src/schema.ts

```diff
@@ -131,44 +164,44 @@ export function build(data: any, opt: QueryConfig = {}): Schema {
   return new Schema(fieldSchemas);
 }
 
-export class Schema {
+export class Schema implements TableSchema {
```

Comment:
Right - cause we are not extending it - but indicating that Schema conforms to the interface. If there is optional values in the interface - we wouldn't expect to always have those optional values.